dev-tools/compose/docker-compose.ddex.yml

```diff
@@ -1,14 +1,106 @@
 version: '3.9'
 
 services:
-  ddex:
-    container_name: ddex
+  ddex-web:
+    container_name: ddex-web
     build:
       context: ${PROJECT_ROOT}
-      dockerfile: ${PROJECT_ROOT}/packages/ddex/Dockerfile
+      dockerfile: ${PROJECT_ROOT}/packages/ddex/webapp/Dockerfile
       args:
-        app_name: ddex
+        app_name: ddex-web
         TURBO_TEAM: '${TURBO_TEAM}'
         TURBO_TOKEN: '${TURBO_TOKEN}'
+    networks:
+      - ddex-network
     profiles:
       - ddex
+
+  ddex-crawler:
+    container_name: ddex-crawler
+    build:
+      context: ${PROJECT_ROOT}/packages/ddex/ingester
+      dockerfile: ${PROJECT_ROOT}/packages/ddex/ingester/Dockerfile
+    environment:
+      - DDEX_MONGODB_URL=mongodb://mongo:mongo@ddex-mongo:27017/ddex?authSource=admin&replicaSet=rs0
+    depends_on:
+      ddex-mongo:
+        condition: service_healthy
+    entrypoint: ./ingester --service crawler
```

Comment:
this makes sense to me for local audius-compose. for ci maybe it's a nit, but would it be better to just build+push the ingester image and set each respective entrypoint in audius-docker-compose like how you're doing here (as opposed to pushing 3 separate images)? not a blocker for merging

Reply:
tru, will add shortly

Reply:
https://github.com/AudiusProject/audius-protocol/pull/7361/commits/045595e0849178697bbfaeec704f703ab0aea132

Reply:
nice 😎